Name: Hide SySToolsMgr Icon
Version: 2.2.4-1
Author: webOS Archive
Description: Hide the SySToolsMgr Icon -- withour removing the app.

--- .orig/media/cryptofs/apps/usr/palm/applications/ca.canucksoftware.systoolsmgr/appinfo.json
+++ /media/cryptofs/apps/usr/palm/applications/ca.canucksoftware.systoolsmgr/appinfo.json
@@ -80,6 +80,7 @@
 		},
 	],
 	"copyright": "&copy; Copyright 2010-2011, Jason Robitaille",
-	"visible": true,
+	"removable": true,
+	"visible": false,
 	"uiRevision": 2
 }
